Cargo.toml

```diff
@@ -11,13 +11,14 @@ git = "https://github.com/dwrensha/capnpc-rust"
 log = "*"
 rand = "*"
 uuid = "*"
+bufstream = "*"
 
 [dependencies.capnp]
 git = "https://github.com/danburkert/capnproto-rust"
 branch = "async"
 
 [dependencies.mio]
-git = "https://github.com/carllerche/mio"
+git = "https://github.com/danburkert/mio"
```

Comment:
Awesome, thanks for citing the PR. :) That makes me feel better. It looks like it's going to get merged.